Makefile

```diff
@@ -83,7 +85,9 @@ envtest:
 test: envtest gotest
 
 .PHONY: env
+.SILENT: env
 env: isclean
-	@echo OPERATOR_NAME=$(OPERATOR_NAME)
-	@echo OPERATOR_NAMESPACE=$(OPERATOR_NAMESPACE)
-	@echo OPERATOR_VERSION=$(VERSION_FULL)
+	echo OPERATOR_NAME=$(OPERATOR_NAME)
```

Comment:
I think you need the `@` on these, otherwise you'll see the `echo OPERATOR_NAME=` command itself, and that will confuse other places which `eval` the output.

Reply:
added `.SILENT: env` to be consistent w/ `osd-operators-registry`

https://github.com/openshift/dedicated-admin-operator/pull/27/files/ddc32ea1ef97f5f00b6d6363f994f67a98f71ad1#diff-b67911656ef5d18c4ae36cb6741b7965R88

Reply:
Oops! I missed that. sorry!